### DISCUSSION TITLE
Police terminal

### DISCUSSION THREAD

**Discussion by MarkTwel** [octothorp-team/public-dobrograd#912] [Предложения]
Предлагаю добавить терминал для экстренных служб. Терминал намного улучшит коммуникацию между офицерами полиции, наверно ещё добавит атмосферу.  Теперь есть вопрос, куда же вставить терминал, я имею виду в сервер gmod или же сделать на сайт. Если же в gmod, то там будет легче реализовать список розыска, выбор автомобиля который в розыске, описание человека, как в F4 или же в гараже, но это может ломать сервер, наверно, но будет легче реализовать чтобы офицер отмечался на смене. Если же сайт, будет сложнее реализовать выбор автомобиля или описание  человека, а так же проверять есть ли офицер на смене и давать ему доступ, но можно будет сделать красивые анимации.

Полезность терминала; улучшение коммуникации, улучшенный список розыска, вызов группы swat (вряд ли), передача обстановки на вызове, передача кодов и уже нормальная система маркировок.

Я думаю что вы опытные разработчики, и то что вы сами решите, да и не факт что одобрят. 
Вот пример, взятый с gta 5. 
![image](https://user-images.githubusercontent.com/98535300/151381105-9d1b2c20-7847-455f-85de-6be4ddd65f90.png)

**Comment by Kradixsit** [comment]
После появления “_системы организаций и персонажей_”, хотят сделать такую систему на Google таблицах.

Сейчас же для удобства BOLO-списка можешь использовать заметки. C > Создать заметку; а затем в рацию спрашиваешь BOLO-Список и когда тебе его скажут, записываешь его в эту записку. (Её можешь подвинуть в любое место, а также растягивать и уменьшать для удобства)

P.S. После появления системы организаций и персонажей уберут муниципальную полицию, так что чтобы увидеть эту систему подавай заявку в ДПД.
